.travis.yml

```
@@ -40,3 +40,5 @@ script:
   - cargo install --debug -f --path .
   - cd ../
   - ./smoke-test.sh
+  # Check that /examples has no changes
+  - if [ -z "$(git status --porcelain)" ]; then echo "/examples is out of date. Please run `./smoke-test.sh` locally and commit the changes"; exit 1; fi
```

Doesn't this need to be '! -z`?
```suggestion
  - if [ ! -z "$(git status --porcelain)" ]; then echo "/examples is out of date. Please run `./smoke-test.sh` locally and commit the changes"; exit 1; fi
```